app/ide-desktop/lib/client/src/naming.ts

```diff
@@ -0,0 +1,15 @@
+// ==============
+// === Naming ===
+// ==============
+
+export function capitalizeFirstLetter(str: string): string {
+    return str.charAt(0).toUpperCase() + str.slice(1)
+}
+
+export function camelToKebabCase(str: string) {
+    return str.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase()
```

Comment:
Does not work with non-ascii letters. :P

Reply:
I know :P We do not need it ;)